### ISSUE TITLE
Please Implement Exoplayer 2.12.0 with ViewPager2 using RecyclerViewAdapter

### ISSUE THREAD

**Issue by nirazv** [MostafaAnter/VideoPlayerInsideRecyclerView#4]
Please Implement Exoplayer 2.12.0 with ViewPager2 using RecyclerViewAdapter and provide github repo

**Comment by Krutika-chotara** [comment]
Hi did you found any solution for viewpager 2?

**Comment by MostafaAnter** [comment]
I have a solution I'll update the repo this week  @nirazv @Krutika-chotara wait for it ;)

**Comment by Krutika-chotara** [comment]
I have implemented properly in view pager 2 but after calling notifydatasetchange player stopped working .

**Comment by AndroidDeveloperMCT** [comment]
Yes notifydatasetchanged() is not working as expected, the player stopped working

**Comment by zakirhussain49** [comment]
is any one found solution : Player not working when we call notifydatasetchanged()

**Comment by NitinPraksash9911** [comment]
Hi All,

Please find my repo which is using updated exo-player 2.17.1
https://github.com/NitinPraksash9911/ExoplayerWithRecyclerView
